coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LongRunningAction.java

```diff
@@ -319,17 +319,18 @@ public boolean restore_state(InputObjectState os, int ot) {
                     }
 
                     if (status == LRAStatus.Active) {
-                        status = LRAStatus.Cancelling; // transition from Active to Cancelling
+                        status = LRAStatus.Cancelling;
```

Comment:
I am not sure about this, does it have a negative effect on how the `this::abortLRA` executes?

Reply:
No, I don't think so. But I'd rather avoid having  a timed out transaction having Active status, cancelling is more appropriate. 
It will be updated anyway in the abortLRA method: https://github.com/jbosstm/lra/blob/main/coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LongRunningAction.java#L1188-L1191

Reply:
Thank you